webapp/app/partials/perf/comparectrl.html

```diff
@@ -0,0 +1,42 @@
+<nav class="navbar navbar-default" role="navigation">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <a class="navbar-brand" href="#">Perfherder Compare Talos</a>
+    </div>
+  </div>
+</nav>
+
+<div class="ph-horizontal-layout">
```

Comment:
Remove the ph-horizontal-layout class, replace with "container-fluid". We don't need flexbox behaviour anymore, and using container-fluid will give us some nice padding.